service/backend/pchain/indexer/testdata/outs/0.json

```diff
@@ -2,7 +2,7 @@
   "id": "4AqeFPxtTW4B5D6oR8gRZTvRKnnqkUWiV6mUNZxjUMbQKYWpi",
   "type": "*blocks.ApricotProposalBlock",
   "parent": "2FUFPVPxbTpKNn39moGSzsmGroYES4NZRdw3mJgNvMkMiMHJ9e",
-  "timestamp": 1600740000000,
+  "timestamp": 1635980695,
```

Comment:
timestamp changes in outs.json are due to the choice of picking index api timestamp rather than banff/proposerVM time